src/methods/method_overloading/hard/q9/GenerateUsername.java

```diff
@@ -0,0 +1,40 @@
+package methods.method_overloading.hard.q9;
+
+/*
+Write a Java Program that receives two first names and ages from the user and passes that information to an
+overloaded method called userName. The method should take the String and int in either order, (name, age) or (age, name),
+and return their username as first_name.age. For example john.34.
```

Comment:
I think you should elaborate on the example (if john and 34 are input, the user name is john.34 )